### PR TITLE
code_8073CF0

### PR DESCRIPTION
**Pull request by DizzyEggg**
WIP

### CODE REVIEW

**Comment by DizzyEggg** [comment]
Ready for review

File: include/constants/dungeon.h [hunk 1]
```diff
@@ -108,4 +108,12 @@ enum DungeonID
     NUM_DUNGEONS
 };
 
+static inline bool8 IsClientOrTeamBase(s32 joinedDungeon)
```
Comment (AnonymousRandomPerson) [inline include/constants/dungeon.h:111]: This function occurs in a number of other places under the name `JoinLocationCannotUseItems`. Perhaps those usages can be combined here too?

Comment (DizzyEggg) [reply include/constants/dungeon.h:111]: Sure, I could do that. What about the name, which one you find better? Or maybe you have a better idea for the name : D

Comment (DizzyEggg) [reply include/constants/dungeon.h:111]: I went with IsClientOrTeamBase

Comment (AnonymousRandomPerson) [reply include/constants/dungeon.h:111]: I originally named `JoinLocationCannotUseItems` for an inline function related to items, but we've now found other occurrences that aren't item-related, like determining whether a Pokémon can gain experience. I'm not sure where `DUNGEON_RESCUE_TEAM_BASE` is used, so I like a literal name like `IsClientOrTeamBase` better as well.

**Review by AnonymousRandomPerson — APPROVED** [review]

**Pull request merged**